src/components/Article/ArticleContents.jsx

```diff
@@ -60,6 +52,17 @@ const ArticleContents = ({ article, locale, modifier = 'clean', t }) => {
           <ArticleFootNotes footNotes={article.metaData?.footnotes} />
         )}
       </LayoutItem>
+      <LayoutItem layout="extend">
+        <ArticleByline
+          licenseBox={<LicenseBox article={article} locale={locale} t={t} />}
+          copyPageUrlLink={article.oembed}
```

Comment:
På denne knappen er det tenkt at det er url-en til sida du står på no som skal kopierast - window.location. Oembed lenka finn ein når ein trykk på retningslinjer for bruk.

Reply:
Det vil altså sei at denne endringa er feil: https://github.com/NDLANO/frontend-packages/commit/0b1eef31bbad929f1ea9bb1de86a1c587a453c38

Reply:
Men gir det egentlig meining med denne knappen? Det er jo det samme som å kopiere urlen i adresselinja. Og for emner og underemner som vises i samme sida vil jo alle ha samme window.location fordi navigeringa skjer via scrolling.

Reply:
UX har talt :): Det kan være det samme som å kopiere urlen i adresselinja, men ikkje dersom man har navigert seg ned til eit underemne, for så å opne "lengre beskrivelse" av overemnet også trykker "kopier lenke". Dersom man gjer det på nederste nivå så vil det sei at det er det same som å kopiere url i nettleser.

Emner og underemner får forskjellige urler når man navigerer(bortsett frå bokmerkenavigasjonen, men det er ei intern navigering).